GooglePay/src/main/java/com/braintreepayments/api/googlepay/GooglePayRequest.kt

```diff
@@ -0,0 +1,437 @@
+package com.braintreepayments.api.googlepay
+
+import android.os.Parcel
+import android.os.Parcelable
+import android.text.TextUtils
+import com.google.android.gms.wallet.ShippingAddressRequirements
+import com.google.android.gms.wallet.TransactionInfo
+import com.google.android.gms.wallet.WalletConstants
+import com.google.android.gms.wallet.WalletConstants.BillingAddressFormat
+import org.json.JSONArray
+import org.json.JSONException
+import org.json.JSONObject
+import java.util.Locale
+
+/**
+ * Represents the parameters that are needed to use the Google Pay API.
+ */
+@SuppressWarnings("TooManyFunctions")
+class GooglePayRequest : Parcelable {
+    /**
+     * Details and the price of the transaction. Required.
+     *
+     * @return See [TransactionInfo].
+     */
+    /**
+     * Details and the price of the transaction. Required.
+     *
+     * @param transactionInfo See [TransactionInfo].
+     */
+    // NEXT_MAJOR_VERSION: allow merchants to set transaction info params individually and build
+    // JSON object under the hood
+    var transactionInfo: TransactionInfo? = null
+    /**
+     * @return If the buyer's email address is required to be returned.
+     */
+    /**
+     * Optional.
+     *
+     * @param emailRequired `true` if the buyer's email address is required to be returned, `false` otherwise.
+     */
```

Comment:
The docs strings on these look like they didn't format correctly for Kotlin - I think we can remove all of the return statements